containers/dart-flutter-android/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "flutter",
```

Comment:
This is the name that will show in the lower left of the tool and in the dropdown in VS Code, so this isn't descriptive enough.  Perhaps "Dart & Flutter" for the name?